Makefile

```diff
@@ -11,7 +11,7 @@ GOVET=$(GOCMD) vet
 GOBUILD=$(GOCMD) build
 GOINSTALL=$(GOCMD) install
 GOCLEAN=$(GOCMD) clean
-GOTEST=$(GOCMD) test
+GOTEST=GOARCH=amd64 CGO_ENABLED=1 $(GOCMD) test
```

Comment:
fix test failed in mac m1

Reply:
Is this flag `-gcflags=all=-l` not requried? It's noted in gomonkey doc: https://github.com/agiledragon/gomonkey#notes

Reply:
> Is this flag `-gcflags=all=-l` not requried? It's noted in gomonkey doc: https://github.com/agiledragon/gomonkey#notes

Makefile has been added before 
```
-gcflags="all=-N -l"
```